src/crypto/algorithms/megolm.js

```diff
@@ -656,6 +665,7 @@ MegolmDecryption.prototype.onRoomKeyEvent = function(event) {
     this._olmDevice.addInboundGroupSession(
         content.room_id, senderKey, sessionId,
         content.session_key, event.getKeysClaimed(),
+        exportFormat,
```

Comment:
well now.

I suspect we actually need *both* keys. This comes back to a conversation the two of us had about verification:

richvdh:
> Alice starts a megolm session, and sends the keys to Bob's device X. Bob hasn't verified Alice's device. Later, Bob logs in on a new device Y, and gets the keys to the megolm session from X
> presumably he now needs to verify both X and Alice's device before that session is considered 'verified' at Y
> to establish some sort of chain of trust

Mjark:
> Yes. Y needs to trust X when X says that Alice's device W is the owner of session S. One way to do that is to verify that X is owned by the same person as Y.
> Separately we then need to verify that device W is owned by Alice.

Currently I don't have anywhere to store two sets of keys; I'm accepting that verification is going to be completely broken for now (because the sender_key on the m.room.encrypted message (W) isn't going to match the claimed key on the session (X)).